UG: Introduce IP Protection (#690)
foundriesio/docs: source/user-guide/ip-protection/ip-protection.rst

.. _ref-ug-ip-protection:

Intellectual Property Protection
================================

Protecting your Intellectual Property (IP) is important for us. Foundries.io™ take best practices to make sure your code is protected when using FoundriesFactory®.

.. seealso::
   :ref:`ref-data-retention`
   privacy company fio site

Here are additional steps you can take to make your FoundriesFactory even more secure and isollate source code access:

* Customers can build their own container images and have FoundriesFactory CI pull those containers by using ``secrets``. See :ref:`ref-private-registries`.

* Customers can use git submodules inside their compose-apps which point at private git repos. With this, only FoundriesFactory CI builders can access the code by using ``secrets``. Foundries.io team members cannot. See :ref:`ug-submodule`.

* In a similar manner, customers can add Yocto Project recipes which reference private git repos, also accessible by FoundriesFactory CI builders with proper ``secrets`` set. See :ref:`ref-ug-private-repo`.

> * Customers can manage source access via their ``Teams`` permissions. A Foundries.io support engineer can be temporarily invited with restrict source code access, so that they can help with other aspects of the FoundriesFactory. See :ref:`ref-team-based-access`.

Reviewer: ```suggestion
* Manage source access via ``Teams`` permissions. A Foundries.io support engineer can be temporarily invited with restrict source code access, so that they can help with other aspects of your Factory. See :ref:`ref-team-based-access`.
```
